modules/ts/misc/run.py

```diff
@@ -879,7 +916,7 @@ def getRunArgs(args):
     run_args = getRunArgs(args[1:] or ['.'])
```

Comment:
`build_path` is still optional parameter with `'.'` default value (script is lauched from the binary directory). So please keep `[build_path]` in help descriptions.